### PR TITLE
fix(makefile): add make target to delete kubernetes generated code

### PR DESCRIPTION
**Pull request by ashishranjan738**
Signed-off-by: Ashish Ranjan <ashishranjan738@gmail.com>

**What this PR does / why we need it**:
This commit adds `make kubegendelete` target for deleting generating code by kubegen and also adds a travis check for this. This will be helpful for catching PR's that introduces inconsistency for generated codes.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `breaking-changes` tag
- [ ] PR messages has breaking changes related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] PR messages has upgrade related information
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -192,6 +192,13 @@ kubegen1:
 # code generation for custom resources
 kubegen: kubegen1 kubegen2
 
+# deletes generated code by codegen
+kubegendelete:
```
Comment (AmitKumarDas) [inline GNUmakefile]: should we include this into kubegen?
```mk
kubegen: kubegendelete kubegen1 kubegen2
```

File: buildscripts/travis-build.sh [hunk 2]
```diff
@@ -55,6 +55,7 @@ printf "\n"
 
 
 echo "Running : make kubegen"
+make kubegendelete
```
Comment (AmitKumarDas) [inline buildscripts/travis-build.sh]: this may not be required if above comment is taken care of

File: GNUmakefile (hunk 1 above)
Comment (ashishranjan738) [reply GNUmakefile]: done

File: buildscripts/travis-build.sh (hunk 2 above)
Comment (ashishranjan738) [reply buildscripts/travis-build.sh]: done

**Pull request merged**